mne_bids/tests/test_meg_bids.py

```diff
@@ -23,6 +24,16 @@
 session_id = '01'
 run = '01'
 task = 'testing'
+acq_a = 'acq1'
+acq_b = 'acq2'
+
+# for windows, shell = True is needed
+# to call npm, bids-validator etc.
+#     see: https://stackoverflow.com/questions/
+#          28891053/run-npm-commands-using-python-subprocess
+shell = False
+if os.name == 'nt':
+    shell = True
```

Comment:
you should just rebase and this will come in your file

Reply:
sorry about that. Hopefully it is fixed

Reply:
nopes, not yet ...